libs/payments/cart/src/lib/cart.service.ts

```diff
@@ -286,13 +293,18 @@ export class CartService {
   /**
    * Fetch a cart from the database by ID
    */
-  async getCart(cartId: string): Promise<WithUpcomingInvoiceCart> {
+  async getCart(
+    cartId: string
+  ): Promise<WithUpcomingInvoiceCart & { metricsOptedOut: boolean }> {
```

Comment:
Could you update the type `WithUpcomingInvoiceCart` with `metricsOptedOut`?

Reply:
Sure - updating the name to `WithInvoiceAndMetricsCart` to reflect this.